readme.md

```diff
@@ -255,6 +255,24 @@ import adapter from '@flopflip/launchdarkly-adapter';
 </ConfigureFlopFlip>;
 ```
 
+You can also pass `render` or `children` as a function to act differently based on the underlying adapter's ready state:
+
+```jsx
+<ConfigureFlopFlip adapter={adapter} adapterArgs={{ clientSideId, user }}>
+  {{isAdapterReady} => isAdapterReady ? <App /> : <LoadingSpinner />}
```

Comment:
This is not 100% recommended as your app should work without flags and not rely on an external service for boot.